packages/sdk-react/src/components/payables/utils/isInvoiceOverdue.ts

```diff
@@ -14,6 +16,7 @@ export const isInvoiceOverdue = (
   return (
     statuses.includes(invoice.status) &&
     invoice.due_date &&
-    new Date(invoice.due_date).getTime() < today.getTime()
+    new Date(invoice.due_date).getTime() < today.getTime() &&
+    new Date(invoice.due_date).getTime() < today.getTime() + 24 * 60 * 60 * 1000
```

Comment:
A comparison a < 10 && a < 20 can be simplified to a < 10. I suggest writing a unit-test to cover all edge-cases. The test will be very simple and it won't take much time to write.

Reply:
Dicussed directly with @chebum , agreed to cover with unit test